wagtail_wordpress_import/test/tests/test_commands.py

```diff
@@ -153,3 +153,10 @@ def test_get_xml_file(self):
             build_xml_stream(xml_tags_fragment="").read()
         )
         self.assertEqual(built_file, cmd.get_xml_file(built_file))
+
+    def test_xml_reduce_generate_stats(self):
```

Comment:
:warning: I have not yet succeeded in running tests locally, so this test has not been ran and I can't say it run successfully.